pkgs/development/mobile/androidenv/compose-android-packages.nix

```diff
@@ -1,8 +1,8 @@
 {requireFile, autoPatchelfHook, pkgs, pkgs_i686, licenseAccepted ? false}:
 
-{ toolsVersion ? "25.2.5"
-, platformToolsVersion ? "29.0.6"
-, buildToolsVersions ? [ "28.0.3" ]
+{ toolsVersion ? "26.1.1"
+, platformToolsVersion ? "30.0.2"
```

Comment:
Another incredibly lame problem: Google removes old platform-tools versions from their XML. I've found that I need to manually add them to the generated packages.